### ISSUE TITLE
Latitude Longitude Table breaks when results_dir is an absolute path

### ISSUE THREAD

**Issue by chengzhuzhang** [ACME-Climate/acme_diags#114]
For example, we run the diags using the following parameter file:
```python
reference_data_path = '/p/cscratch/acme/data/obs_for_acme_diags/'
test_data_path = '/p/cscratch/acme/data/test_model_data_for_acme_diags/'

test_name = '20161118.beta0.FC5COSP.ne30_ne30.edison'

backend = 'mpl'
results_dir = '/export_2/zhang40/acme_diags/acme_diags/tests_table_6'
multiprocessing = True
num_workers = 24
seasons = ['ANN','DJF']
sets =['lat_lon']
```

and if we move it to `/p/cscratch/acme/www/`, the lat-loa tables don't work. However, if we set `results_dir = 'tests_table_6'` and then move it to the aforementioned location, it will actually work this time.

@zshaheen Would you please fix this before releasing 1.1.1?

**Comment by zshaheen** [comment]
@chengzhuzhang Yes Jill, I'll fix this right away. I am very, very sorry for these issues. I promise that I will never, ever let this happen again.

**Comment by chengzhuzhang** [comment]
@zshaheen No worries. Glad that we figured it out.

**Comment by zshaheen** [comment]
Fixed in 8312e01595a7d76fb88eff9a3147c14c8c4bdd55
